### ISSUE TITLE
[openblas] Unable to build

### ISSUE THREAD

**Issue by uberhacker** [skycocker/chromebrew#2229]
Build on i686:
```
ar: strmm_kernel_LN.o: No such file or directory
make[1]: *** [../Makefile.tail:40: libs] Error 1
make[1]: Leaving directory '/usr/local/tmp/crew/v0.2.20.zip.dir/OpenBLAS-0.2.20/kernel'
make: *** [Makefile:139: libs] Error 1
openblas failed to build: `make V=0 -j1` exited with 2
```

**Comment by uberhacker** [comment]
Build on x86_64:
```
Makefile:128: *** OpenBLAS: Detecting CPU failed. Please set TARGET explicitly, e.g. make TARGET=your_cpu_target. Please read README for the detail..  Stop.
openblas failed to build: `make V=0 -j2` exited with 2
```

**Comment by lovedheart** [comment]
compile it using generic option?

**Comment by uberhacker** [comment]
Fixed by #2250.
